stable/jenkins/values.yaml

```diff
@@ -17,7 +17,10 @@ Master:
 # JavaOpts: "-Xms512m -Xmx512m"
   ServicePort: 8080
 # For minikube, set this to NodePort, elsewhere use LoadBalancer
+# Use ClusterIP if your setup includes ingress controller
   ServiceType: LoadBalancer
+# Used to create Ingress record (should used with ServiceType: ClusterIP)
+  HostName: ~
```

Comment:
I would prefer to have this key uncommeted, but empty. But if there is a convention to comment out empty keys - lets stick to it.